avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -904,6 +945,11 @@ def updateFieldsC(cfg, particles, dem, fields):
   fields['ppr'] = np.asarray(PP)
   fields['pfd'] = np.asarray(PFD)
 
+  if resetPVals == 1:
+    fields['pfv'] = np.asarray(VBilinear)
+    fields['ppr'] = np.asarray(PBilinear)
+    fields['pfd'] = np.asarray(FDBilinear)
```

Comment:
Cant this be done outside? in the com1DFA function, at the end of the ini steps?

Reply:
moved to com1DFA